### PR TITLE
Docs: Add section on 'Publish to Branch' feature

### PR DESCRIPTION
**Pull request by luzpaz**
also incl. screenshot

### CODE REVIEW

**Comment by luzpaz** [comment]
Should be merged after #6

File: docs/index.md [hunk 1]
```diff
@@ -19,3 +19,27 @@ configure_mod_raytracing()
...
+
+In the the screenshot above the first field labeled **`FreeCAD commit/branch/tag`** is able to recieve the following inputs:  
+
+* **`commit`** this is a specific git hash of an open Pull Request in FreeCAD/FreeCAD  
+* **`branch`** is for special FreeCAD/FreeCAD branches one would like to compile. Simply adding the name of said branch is enough. For example: 'Toponaming' (note branch should exist first).
+* **`tag`** is for building snaps from a specific FreeCAD/FreeCAD release tag. 
+
+The secound field in the above screenshot **`Snapcraft branch`** is simply the unique build name you can grant to this specific build. 
+
+Then simply pressing the **Run workflow** button will trigger said build. When build is complete, the method to run this build is simple. In the CLI invoke:
```
Comment (ppd) [inline docs/index.md:37]:
```suggestion
Then simply pressing the **Run workflow** button will trigger said build. When build is complete, the method to install this build is simple. In the CLI invoke:  
```

File: docs/index.md [hunk 2]
```diff
@@ -19,3 +19,27 @@ configure_mod_raytracing()
...
 
 **Convention**: One function per workbench, don't overwrite the user's settings.
+
+## Publishing to Branch
+
+There is an option to build Snaps from experimental commits, branches, or even git release tags. 
+
+![Publish-to-Branch](assets/images/Snap-Publish-to-Branch-screenshot.png)
+
+In the the screenshot above the first field labeled **`FreeCAD commit/branch/tag`** is able to recieve the following inputs:
```
Comment (ppd) [inline docs/index.md:29]:
```suggestion
In the the screenshot above the first field labeled **`FreeCAD commit/branch/tag`** is able to receive the following inputs:  
```

File: docs/index.md [hunk 3]
```diff
@@ -19,3 +19,27 @@ configure_mod_raytracing()
...
+* **`tag`** is for building snaps from a specific FreeCAD/FreeCAD release tag. 
+
+The secound field in the above screenshot **`Snapcraft branch`** is simply the unique build name you can grant to this specific build. 
+
+Then simply pressing the **Run workflow** button will trigger said build. When build is complete, the method to run this build is simple. In the CLI invoke:  
+```shell
+snap install freecad --channel <experimental-build-name>
+```
+
+When you're done testing the build, to return to the original build you were running just swap out the build name; for example: `stable` or `edge`
```
Comment (ppd) [inline docs/index.md:42]:
```suggestion
When you're done testing the build, to return to the original build you were running just swap out the build name and refresh; for example: `stable` or `edge` 
```

File: docs/index.md [hunk 4]
```diff
@@ -19,3 +19,27 @@ configure_mod_raytracing()
...
+The secound field in the above screenshot **`Snapcraft branch`** is simply the unique build name you can grant to this specific build. 
+
+Then simply pressing the **Run workflow** button will trigger said build. When build is complete, the method to run this build is simple. In the CLI invoke:  
+```shell
+snap install freecad --channel <experimental-build-name>
+```
+
+When you're done testing the build, to return to the original build you were running just swap out the build name; for example: `stable` or `edge` 
+```shell
+snap install freecad --channel edge
```
Comment (ppd) [inline docs/index.md:44]:
```suggestion
snap refresh freecad --channel edge
```

**Review by ppd — COMMENTED** [review]
Excellent stuff. Let's merge this first and then the other PR.

**Comment by ppd** [comment]
> Should be merged after #6

I'd think the other way around. The target of this PR is the docs branch. Therefore, the other PR will contain this stuff, too.

**Pull request merged**